pkg/management/component.go

```diff
@@ -209,6 +211,10 @@ type Component interface {
 	UnlinkShadowUser(ctx context.Context, realmName string, userID string, provider string) error
 
 	GetIdentityProviders(ctx context.Context, realmName string) ([]api.IdentityProviderRepresentation, error)
+
+	GetThemeConfiguration(ctx context.Context, realmName string) (api.ThemeConfiguration, error)
```

Comment:
General question, did we ask for endpoint in management API ? or did we discuss about that ?